server/chat-plugins/datasearch.js

```diff
@@ -419,6 +422,29 @@ function runDexsearch(target, cmd, canAll, message) {
 				continue;
 			}
 
+			if (target.split(' ')[0] === 'asc' || target.split(' ')[0] === 'desc') {
```

Comment:
Incredibly inefficient.

```suggestion
			if (target.startsWith('asc ') || target.startsWith('desc ')) {
```

Also I think the standard is put the sort order after the key.

Reply:
Like, `spe asc` or `spe+`?

Reply:
`spe asc`, but `spe+` might not be a bad idea either.

Reply:
Changed to have both of em